tests/auto/instantiatetypes/tst_instantiatetypes.cpp

```diff
@@ -42,15 +43,20 @@ tst_InstantiateTypes::tst_InstantiateTypes()
     : QObject()
     , m_qmlEngine(new QQmlEngine(this))
 {
+
+}
+
+void tst_InstantiateTypes::initTestCase()
+{
     // Add extensionplugin import path
-    m_qmlEngine->addImportPath(QStringLiteral("%1/../../../qml").arg(QCoreApplication::applicationDirPath()));
+    m_qmlEngine->addImportPath(QStringLiteral("%1/../../../qml")
```

Comment:
I'd say add a QVERIFY that the path actually exists before passing it to addImportPath.